pytorch_trainer/trainer/base_trainer.py

```diff
@@ -165,5 +165,9 @@ def register_callback(self,
         # self._register_hook(config.CUSTOM_HOOK)
 
     def data_to_device(self, data):
-        inputs, labels = data
-        return inputs.to(self.device), labels.to(self.device)
+        # TODO: place to hook
+        if isinstance(data, dict):
+            return {k: v.to(self.device) for k, v in data.items()}
+        else:
```

Comment:
If we use dict output in dataloader, just keep dict version only.

Reply:
ok